internal/nightfallconfig/nightfall_config.go

```diff
@@ -53,6 +54,6 @@ func GetNightfallConfigFile(workspacePath, fileName string) (*NightfallConfigFil
 	if nightfallConfig.MaxNumberRoutines == 0 {
 		nightfallConfig.MaxNumberRoutines = defaultMaxNumberRoutines
 	}
-
+	nightfallConfig.FileExclusionList = append(nightfallConfig.FileExclusionList, nightfallConfigFilename)
```

Comment:
Could it be possible for `nightfallConfig.FileExclusionList` to be `nil`? if so would this append fail?

Reply:
The append will still work. The `fileExclusionList` in our test config is also currently nil, so this should be fine.

https://play.golang.org/p/CvAWyyZQpr3

Reply:
Cool!